### PR TITLE
transparent security post

### PR DESCRIPTION
**Pull request by chayim**

### CODE REVIEW

File: blogposts/2020/improving-security-through-real-transparency.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+## Where we are today
+
+At Sourcegraph, we’re starting down the path to Transparent Security.  We’ve conducted our most recent penetration test, with a trusted third-party. You can download those results [here](https://drive.google.com/file/d/14ZIDhAql26THcLlgCbRhbhez9UXPeiGu/view?usp=sharing).  We maintain a [private repository](https://github.com/sourcegraph/security-issues) for security issues, but [publicly discuss our roadmap and backlog](https://github.com/orgs/sourcegraph/projects?query=is%3Aopen+Security). We disclose security vulnerabilities as they’re patched, and update our progress on the security front.  In some cases, we’re [public](https://github.com/sourcegraph/sourcegraph/issues/12263) with the [issues](https://github.com/sourcegraph/sourcegraph/issues/12269) we’re addressing, but at the same time, we’re not fully transparent —  yet.
```
Comment (chrispine) [inline blogposts/2020/improving-security-through-real-transparency.md:23]:
```suggestion
At Sourcegraph, we’re starting down the path to Transparent Security.  We’ve conducted our most recent penetration test with a trusted third-party. You can download those results [here](https://drive.google.com/file/d/14ZIDhAql26THcLlgCbRhbhez9UXPeiGu/view?usp=sharing).  We maintain a [private repository](https://github.com/sourcegraph/security-issues) for security issues, but [publicly discuss our roadmap and backlog](https://github.com/orgs/sourcegraph/projects?query=is%3Aopen+Security). We disclose security vulnerabilities as they’re patched, and update our progress on the security front.  In some cases, we’re [public](https://github.com/sourcegraph/sourcegraph/issues/12263) with the [issues](https://github.com/sourcegraph/sourcegraph/issues/12269) we’re addressing, but at the same time, we’re not fully transparent —  yet.
```

File: blogposts/2020/improving-security-through-real-transparency.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+## Where we want to be tomorrow
+
+Security should never be focused solely on responsible disclosure and responding in reactive mode. It’s about proactively doing what’s right, to protect data. It’s about professionals working together for the common good. A transparent approach means we’re collaborating on that proactivity with the public, not just the Sourcegraph team. You, dear reader can help be that value, that second set of eyes! The next step for Sourcegraph is to run ongoing vulnerability scans, and publish them publicly. We’ll begin by doing so after we’ve addressed some of the vulnerabilities, and over time these reports will be provided closer and closer to real-time.  We’re in the midst of documenting our security policies, which we will post publicly in our handbook. This is only the beginning.
```
Comment (chrispine) [inline blogposts/2020/improving-security-through-real-transparency.md:27]:
```suggestion
Security should never be focused solely on responsible disclosure and responding in reactive mode. It’s about proactively doing what’s right, to protect data. It’s about professionals working together for the common good. A transparent approach means we’re collaborating on that proactivity with the public, not just the Sourcegraph team. You, dear reader, can help be that value, that second set of eyes! The next step for Sourcegraph is to run ongoing vulnerability scans, and publish them publicly. We’ll begin by doing so after we’ve addressed some of the vulnerabilities, and over time these reports will be provided closer and closer to real-time.  We’re in the midst of documenting our security policies, which we will post publicly in our handbook. This is only the beginning.
```

**Review by chrispine — APPROVED** [review]
Nice! I had only the tiniest of nits to pick. Thanks!

**Comment by chayim** [comment]
> Nice! I had only the tiniest of nits to pick. Thanks!

@chrispine I really appreciate these, thank you!

**Review by lorihey — APPROVED** [review]
Made a few minor edits, and it's good to go now. Sorry for the delay.

**Pull request merged**